api/spec/src/customer.tsp

```diff
@@ -113,6 +113,7 @@ interface Customers {
   @summary("List customer subscriptions")
   listCustomerSubscriptions(
     @path customerId: ULID,
+    @query expandToView?: boolean = false,
```

Comment:
Please follow the patterns used at other endpoints with `expand` and an enum. If not specified only the `id` is returned in the object.